server/views/referrals/courseParticipations/details.njk

```diff
@@ -0,0 +1,172 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% extends "../../partials/layout.njk" %}
+
+{% block personBanner %}
+  {% include "../_personBanner.njk" %}
+{% endblock personBanner %}
+
+{% block backLink %}
+  {{ govukBackLink({
+    text: "Back",
+    href: referPaths.programmeHistory.index({ referralId: referralId })
+  }) }}
+{% endblock backLink %}
+
+{% set communitySettingHtml %}
+{{ govukInput({
+  id: "communityLocation",
+  name: "setting[location]",
```

Comment:
May need to rethink the naming of this one when it comes to implementing the form submission as it shares the same name with the `custodyLocation` input.  Every chance that a user could input values in all of the conditionally displayed inputs and they would still get submitted so could end up with something like  `{ setting: { location: ['Value1', 'Value2'] } }.

Reply:
Yeah maybe we should call this `setting[communityLocation]` and the one below `setting[custodyLocation]`. Or we could handle this in the controller/service to make sure we're picking up the right value, but that might be a bit more complicated.

Reply:
No unfortunately not.  Having the same html doesn't make a difference either.  Will work out what's best while I'm adding the form functionality today, so will possibly be tweaked in the next PR.